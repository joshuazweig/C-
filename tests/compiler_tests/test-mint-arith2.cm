int main()
{
  stone x;
  stone p;

  mint m;

  x = "15352395";
  p = "65537"; 

  m = <x + x, p>;

  print_mint(m);

  return 0;
}
