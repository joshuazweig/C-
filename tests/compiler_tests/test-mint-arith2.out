33474
65537
